Documentation~/modes.md

```diff
@@ -1,42 +1,20 @@
-# Edit modes (Object vs Element)
+# Edit modes and active contexts
 
-ProBuilder uses **Edit modes** to define what you are selecting and editing.
+There are two active contexts when editing in the Unity Editor:
```

Comment:
This isn't necessarily true: it depends on what packages you have installed. If you have the Splines package installed, for example, you would have the Splines context in your project. In the future, there'll be even more.

Reply:
I think what we want to say is that ProBuilder tools use (or can be used _in_, rather) two contexts.

Reply:
```suggestion
You can use ProBuilder tools in two contexts:
```

Reply:
It would be really nice if `tool contexts` could link to a documentation concept page about tool contexts. 
```suggestion
You can use ProBuilder tools in two tool contexts:
```